docs/websites/apache-tips-and-tricks/apache-new-file-handler.md

```diff
@@ -0,0 +1,589 @@
+---
+author:
+  name: Chuck Jungmann
+  email: chuck@cpjj.net
+description: 'Using a CGI or FastCGI Script as a File Handler'
+keywords: 'apache,cgi,fastcgi,AddHandler,SetHandler,AddType,mod_fastcgi'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+modified: 'Tuesday, June 22nd, 2016'
+modified_by:
+  name: Chuck Jungmann
+published: 'Tuesday, June 22nd, 2016'
+title: 'Configuring a Script Intepreter for Apache'
+external_resources:
+ - '[Apache HTTP Server Version 2.4 Documentation](https://httpd.apache.org/docs/2.4/)'
+ - '[Module mod_fastcgi](https://htmlpreview.github.io/?https://github.com/FastCGI-Backups/mod_fastcgi/blob/master/docs/mod_fastcgi.html)'
+ - '[Archive of FastCGI.com](https://github.com/FastCGI-Archives/FastCGI.com/blob/master/README.md)'
+ - '[CGI Environment Variables](http://cgi101.com/learn/ch3/text.html)'
+ - '[Debugging with GDB](http://www.delorie.com/gnu/docs/gdb/gdb_toc.html)'
+---
+
+I have written a script interpreter that I want Apache to use when
+files with a certain extension are called.  I could not find any help
+doing this online, despite how easy it turned out to be.  I have written
+this guide to help others succeed where I struggled to understand.
+
+This guide will use a simple CGI script to demonstrate how to configure
+Apache to use the script as a file handler when an appropriate file is
+called.  Then it will show how to convert the CGI script to a FastCGI
+script for performance reasons, and how to modify the configuration to
+use the FastCGI version instead of the CGI script.
+
+The main sections are arranged thus:
+- [Prerequisites](#Prerequisites)
+- [The CGI Script](#TheCGIScript)
+- [Configuring Apache to Use a File Handler](#ConfiguringApache)
+- [Test CGI File Handler](#TestCGIHandler)
+- [Upgrading CGI to FastCGI](#UpgradeToFastCGI)
+- [Test FastCGI File Handler](#TestFastCGIHandler)
+- [Reading the Target File](#ReadingTheTargetFile)
+- [CGI/FastCGI Programming Suggestions](#ScriptProgrammingSuggestions)
+
+## Prerequisites                            {#Prerequisites}
```

Comment:
You can also get rid of ` {#Prerequisites}` as anchors will be created automatically.